src/ORM/Connect/DBQueryBuilder.php

```diff
@@ -242,9 +242,25 @@ public function buildUpdateFragment(SQLUpdate $query, array &$parameters)
     public function buildFromFragment(SQLConditionalExpression $query, array &$parameters)
     {
         $from = $query->getJoins($joinParameters);
+        $tables = [];
+        $joins = [];
+
+        // E.g. a naive "Select 1" statement is valid SQL
+        if (empty($from)) {
+            return '';
+        }
+
+        foreach ($from as $joinOrTable) {
+            if (preg_match(SQLConditionalExpression::JOIN_REGEX, $joinOrTable)) {
```

Comment:
A const seems fine to me... it's constant, as in it won't ever return a different value. But I'll swap it to a method to avoid unnecessary review ping pong.

Reply:
Done